### PR TITLE
Refactor/color table

### PR DESCRIPTION
**Pull request by marvinLaubenstein**
Reviewer: Josha

### CODE REVIEW

**Comment by marvinLaubenstein** [comment]
⚠️ package.json was pushed only for the review. Please merge only after remove

File: package.json [hunk 1]
```diff
@@ -33,5 +33,8 @@
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
+  "dependencies": {
+    "@telekom/design-tokens": "^1.0.0-beta.1"
```
Comment (acstll) [inline package.json:38]: @marvinLaubenstein this can be removed 🤓

**Comment by marvinLaubenstein** [comment]
@acstll do you see any problems if I merge this ? Especially in relation to the manual version change 🤨

**Comment by acstll** [comment]
> @acstll do you see any problems if I merge this ? Especially in relation to the manual version change 🤨

no problem at all, please go ahead — the manual version change was just needed 🤓

Thank you!  🚀

**Pull request merged**